Emby.Server.Implementations/Session/SessionWebSocketListener.cs

```diff
@@ -55,6 +94,7 @@ void _serverManager_WebSocketConnected(object sender, GenericEventArgs<IWebSocke
             if (session != null)
             {
                 EnsureController(session, e.Argument);
+                var task = KeepAliveWebSocket(e.Argument);
```

Comment:
where `task` is used?

Reply:
Nowhere. I had to add it in order to suppress the `Because this call is not awaited, execution of the current method continues before the call is completed` issue on Sonar. `In that case [no await for async call], you can suppress the warning by assigning the task result of the call to a variable.`

Should `KeepAliveWebSocket` be awaited instead?
Actually, now that I'm seeing the other comment bellow regarding the shown warning, I'm considering making `KeepAliveWebSocket` non-async, then letting the rather rare exception returned when trying to send a message be ignored and finally letting the watcher handle the WebSocket that will eventually time out.

Reply:
I think the better way is really making `KeepAliveWebSocket` non-async.